CardIOSample/CardIOSample/DisplaySVViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  DisplaySVViewController.swift
+//  CardIOSample
+//
+//  Created by Philip Sawyer on 5/26/16.
+//  Copyright © 2016 Philip Sawyer. All rights reserved.
+//
+
+import UIKit
+
+class DisplaySVViewController: UIViewController, CardIOViewDelegate {
+    
+    @IBOutlet weak var scanCardButton: UIButton!
+    @IBOutlet weak var resultsLabel: UILabel!
+    @IBOutlet weak var cancelButton: UIButton!
+    
+    var cardIOView : CardIOView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = "CardIO as SubView"
```

Comment:
Take a look at https://github.com/willowtreeapps/swift-style-guide

House style suggests not using self when not needed. https://github.com/willowtreeapps/swift-style-guide#use-of-self